client/entrypoints/express-checkout/index.js

```diff
@@ -317,6 +372,17 @@ jQuery( function ( $ ) {
 			);
 
 			eceButton.on( 'confirm', async ( event ) => {
+				if (
+					getExpressCheckoutData( 'is_product_page' ) &&
+					wcStripeECE.isAddToCartSuccessful === false
```

Comment:
Is the strict check necessary? Like, is it possible for isAddToCartSuccessful to be of a different type?

Reply:
It's probably not necessary. Just thought to add it in case there is a code path that didn't set `wcStripeECE.isAddToCartSuccessful`, in which case it would be `undefined` and I want the behavior to proceed as it would normally.